ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/tabsontop/v1/tabsontop/_cq_editConfig.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
```

Comment:
should not be required if extended from `panelcontainer`

Reply:
done